bash_functions.sh

```diff
@@ -47,7 +47,7 @@ prepare_configs() {
     installConfigs
    
     if [ ! -f "${setupVars}" ]; then
-        touch "${setupVars}"
+        install /dev/null "${setupVars}"
```

Comment:
```suggestion
        install -m 644/dev/null "${setupVars}"
```
?

Reply:
Or `install -m 644 -o pihole -g pihole /dev/null "${setupVars}"`?

Reply:
Maaaaybe. I was only going by it having a plain `touch` with no mode change before. Do you think we should set the mode here?